packages/adp-tooling/test/unit/preview/adp-preview.test.ts

```diff
@@ -161,4 +162,37 @@ describe('AdaptationProject', () => {
             expect(next).toBeCalled();
         });
     });
+    describe('addApis', () => {
+        let server!: SuperTest<Test>;
+        beforeAll(async () => {
+            const adp = new AdpPreview(
+                {
+                    target: {
+                        url: backend
+                    }
+                },
+                mockProject as unknown as ReaderCollection,
+                logger
+            );
+
+            const app = express();
+            adp.addApis(app);
+            server = await supertest(app);
```

Comment:
Please remove `await`, `supertest` is not async

Reply:
c92882d